corehq/apps/app_manager/models.py

```diff
@@ -4781,6 +4781,9 @@ def validate_detail_screen_field(field):
 
 class SavedAppBuild(ApplicationBase):
 
+    # Whether the application has had data submitted against it
+    has_submissions = BooleanProperty(default=None)
```

Comment:
default should actually be false and if the property is missing then we consider it "unknown" and analyze it. this avoids a migration